packages/leva/src/components/ButtonGroup/StyledButtonGroup.tsx

```diff
@@ -0,0 +1,8 @@
+import { styled } from '../../styles'
+
+export const StyledButtonGroup = styled('div', {
+  $leva__flex: '',
+  display: 'flex',
+  justifyContent: 'flex-end',
+  gap: '5px',
```

Comment:
There's a `$leva__rowGap` stitches property that you can use here for consistency.

```js
gap: '$leva__rowGap'
```